descriptors/ModuleDescriptor-template.json

```diff
@@ -57,7 +57,8 @@
         {
           "methods": ["DELETE"],
           "pathPattern": "/records-editor/records/{id}",
-          "permissionsRequired": ["records-editor.records.item.delete"]
+          "permissionsRequired": ["records-editor.records.item.delete"],
+          "modulePermissions": ["change-manager.parsedrecords.get"]
```

Comment:
"change-manager.jobexecutions.post",
            "change-manager.jobexecutions.put",
            "change-manager.records.post"
            
This permissions also should be added